From: Fiona Klute <fiona.klute@gmx.de>
Date: Wed, 26 Aug 2026 19:44:25 +0200
Subject: fix(shutdown): make "timeout" call busybox-compatible

Busybox' "timeout" does not support long-form "--signal". Use short "-s"
instead.

Part of: https://github.com/dracut-ng/dracut/issues/2437
Origin: upstream, https://github.com/dracut-ng/dracut/commit/8f5c7dd81bc039f8388c46ea2e8c8628aaffed03
---
 modules.d/86shutdown/shutdown.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules.d/86shutdown/shutdown.sh b/modules.d/86shutdown/shutdown.sh
index fd15ab5..0a1e915 100755
--- a/modules.d/86shutdown/shutdown.sh
+++ b/modules.d/86shutdown/shutdown.sh
@@ -74,7 +74,7 @@ umount_a() {
         # be produced by the shell.
         (
             set +m
-            timeout --signal=KILL "$_umount_timeout" umount "$mp"
+            timeout -s KILL "$_umount_timeout" umount "$mp"
         ) 2>&7
         local ret=$?
         if [ $ret -eq 0 ]; then
